pkg/operator/deploy/create.go

```diff
@@ -0,0 +1,321 @@
+package deploy
+
+import (
+	"fmt"
+	"path/filepath"
+
+	meteringv1 "github.com/operator-framework/operator-metering/pkg/apis/metering/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/api/core/v1"
+)
+
+func (deploy *Deployer) createNamespace() error {
+	_, err := deploy.Client.CoreV1().Namespaces().Get(deploy.Namespace, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		namespaceObj := &v1.Namespace{
```

Comment:
This can be done as a follow up, perhaps add a TODO comment for this:

We'll want to annotate the namespace we create the same way that `openshift-install.sh` when we're doing an Openshift install, and update the namespace with the annotation if the namespace already exists.